generators/server/templates/src/main/java/package/service/UserService.java.ejs

```diff
@@ -644,11 +644,11 @@ public class UserService {
             .doOnNext(user -> log.debug("Changed Information for User: {}", user))
             .then();
   <%_ } else { _%>
-    <%_ if (databaseTypeMongodb || databaseTypeCouchbase || databaseTypeNeo4j || databaseTypeCassandra) { _%>
+    <%_ if (databaseTypeSql || databaseTypeMongodb || databaseTypeCouchbase || databaseTypeNeo4j || databaseTypeCassandra) { _%>
```

Comment:
Every db isn’t it?
```suggestion
```

Reply:
Probably then. Don't know, if there is another type of DB where it does not belong... I'll get rid of the condition.

Reply:
@OmarHawk You missed also this condition